login-workflow/src/subScreens/CreateAccount.tsx

```diff
@@ -83,17 +84,22 @@ function isValidEmail(text: string): boolean {
 export const CreateAccount: React.FC<CreateAccountProps> = (props) => {
     const theme = useTheme(props.theme);
     const [emailInput, setEmailInput] = React.useState('');
+    const [emailError, setEmailError] = React.useState('');
+    const [validateEmailOnChange, setValidateEmailOnChange] = React.useState(false);
     const { t } = useLanguageLocale();
 
     const containerStyles = makeContainerStyles(theme);
     const styles = makeStyles();
-    const onChangeText = (text: string): void => {
-        setEmailInput(text);
-        const validEmailOrEmpty = isValidEmail(text) ? text : '';
-        props.onEmailChanged(validEmailOrEmpty);
-    };
 
-    const showEmailError = emailInput.length !== 0 && !isValidEmail(emailInput);
+    const validateEmail = (text?: string): void => {
+        if (text ? !isValidEmail(text) : !isValidEmail(emailInput)) {
+            setEmailError(t('MESSAGES.EMAIL_ENTRY_ERROR'));
+            props.onEmailChanged('');
+        } else {
+            setEmailError('');
+            text ? props.onEmailChanged(text) : props.onEmailChanged(emailInput);
```

Comment:
This is where the linting issue is. The ternary operator is usually used for variable assignment, not function calls like this. I would just use a simple if statement here:
```suggestion
            if( text){ props.onEmailChanged(text); }
            else { props.onEmailChanged(emailInput); }
```